syntax = "proto3";

package syft.lib.syfertext;

import "proto/lib/syfertext/data/units/attribute_values.proto";
import "proto/lib/syfertext/data/units/token_meta.proto";

message TextDoc {
    bytes uuid = 1;
    repeated TokenMeta token_metas = 2;
    map<string, AttributeValues> attributes = 3;
}